### PR TITLE
PLATFORM-3230: liftigniter - switch to petametrics to use https

### PR DESCRIPTION
**Pull request by themech**
fandommetrics was a CNAME redirect to petametrics and was not covered by https cert. Switch back to petametrics and it allows us to use https.

### CODE REVIEW

**Review by rybmat — APPROVED** [review]

**Review by Grunny — APPROVED** [review]

File: extensions/wikia/Recirculation/js/trackers/liftigniter.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (function(w,d,s,p,v,e,r) {w['$igniter_var']=v;w[v]=w[v]||function(){(w[v].q=w[v].q||[]).push(
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/js/trackers/liftigniter.js:1]: MAJOR Function has 7 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AAmdExcludedFunctionParameterLimit)
MINOR Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)

File: extensions/wikia/Recirculation/js/trackers/liftigniter.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 arguments)};w[v].l=1*new Date();e=d.createElement(s),r=d.getElementsByTagName(s)[0];e.async=1;
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/js/trackers/liftigniter.js:2]: MINOR Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 118 issues
* ![MAJOR][MAJOR] 89 major
* ![MINOR][MINOR] 16 minor
* ![INFO][INFO] 13 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiFactory.php](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php): File "WikiFactory.php" has 3480 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS104)
1. ![MAJOR][MAJOR] [WikiFactory.php#L27](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php#L27): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L49](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php#L49): Class "WikiFactory" has 77 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [WikiFactory.php#L258](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php#L258): Reduce the number of returns of this function 6, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. ![MAJOR][MAJOR] [WikiFactory.php#L262](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php#L262): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L267](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php#L267): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L361](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php#L361): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L407](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php#L407): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L490](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php#L490): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L571](https://github.com/Wikia/app/blob/634d704ab5fd4681f54da1ceda949ae7f3ecd7e8/extensions/wikia/WikiFactory/WikiFactory.php#L571): The Cyclomatic Complexity of this function "setVarById" is 30 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: extensions/wikia/WikiFactory/WikiFactory.php [hunk 3]
```diff
@@ -1226,6 +1226,8 @@ public static function getCurrentStagingHost( $dbName='', $default='', $host = n
 	static public function getLocalEnvURL( $url, $forcedEnv = null ) {
```
Comment (WikiaTech) [inline extensions/wikia/WikiFactory/WikiFactory.php]: MAJOR Reduce the number of returns of this function 8, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

File: extensions/wikia/WikiFactory/WikiFactory.php [hunk 4]
```diff
@@ -1226,6 +1226,8 @@ public static function getCurrentStagingHost( $dbName='', $default='', $host = n
...
 		global $wgWikiaEnvironment, $wgWikiaBaseDomain, $wgDevDomain;
 
+        Wikia::log( __METHOD__, "", "WikiFactory is not used." );
```
Comment (WikiaTech) [inline extensions/wikia/WikiFactory/WikiFactory.php]: MAJOR Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 119 issues
* ![MAJOR][MAJOR] 90 major
* ![MINOR][MINOR] 16 minor
* ![INFO][INFO] 13 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiFactory.php](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php): File "WikiFactory.php" has 3488 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS104)
1. ![MAJOR][MAJOR] [WikiFactory.php#L27](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php#L27): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L49](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php#L49): Class "WikiFactory" has 77 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [WikiFactory.php#L258](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php#L258): Reduce the number of returns of this function 6, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. ![MAJOR][MAJOR] [WikiFactory.php#L262](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php#L262): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L267](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php#L267): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L361](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php#L361): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L407](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php#L407): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L490](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php#L490): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L571](https://github.com/Wikia/app/blob/2b0df03c265c88e88a32fd453022c806553e44d2/extensions/wikia/WikiFactory/WikiFactory.php#L571): The Cyclomatic Complexity of this function "setVarById" is 30 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: extensions/wikia/Recirculation/js/trackers/liftigniter.js (hunk 1 above)
Comment (WikiaTech) [inline extensions/wikia/Recirculation/js/trackers/liftigniter.js:1]: MAJOR Function has 7 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AAmdExcludedFunctionParameterLimit)

[project-id]: wikia:app
MINOR Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)

[project-id]: wikia:app

File: extensions/wikia/Recirculation/js/trackers/liftigniter.js (hunk 2 above)
Comment (WikiaTech) [inline extensions/wikia/Recirculation/js/trackers/liftigniter.js:2]: MINOR Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 118 issues
* ![MAJOR][MAJOR] 89 major
* ![MINOR][MINOR] 16 minor
* ![INFO][INFO] 13 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiFactory.php](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php): File "WikiFactory.php" has 3480 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS104)
1. ![MAJOR][MAJOR] [WikiFactory.php#L27](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php#L27): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L49](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php#L49): Class "WikiFactory" has 77 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [WikiFactory.php#L258](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php#L258): Reduce the number of returns of this function 6, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. ![MAJOR][MAJOR] [WikiFactory.php#L262](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php#L262): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L267](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php#L267): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L361](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php#L361): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L407](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php#L407): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L490](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php#L490): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L571](https://github.com/Wikia/app/blob/a171b03e5f5329f8187651b47732b0b12fccdfc1/extensions/wikia/WikiFactory/WikiFactory.php#L571): The Cyclomatic Complexity of this function "setVarById" is 30 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 118 issues
* ![MAJOR][MAJOR] 89 major
* ![MINOR][MINOR] 16 minor
* ![INFO][INFO] 13 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiFactory.php](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php): File "WikiFactory.php" has 3469 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS104)
1. ![MAJOR][MAJOR] [WikiFactory.php#L27](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php#L27): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L49](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php#L49): Class "WikiFactory" has 77 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [WikiFactory.php#L258](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php#L258): Reduce the number of returns of this function 6, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. ![MAJOR][MAJOR] [WikiFactory.php#L262](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php#L262): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L267](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php#L267): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L361](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php#L361): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L407](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php#L407): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L490](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php#L490): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L571](https://github.com/Wikia/app/blob/6cf515a1c344a77f5864d413b3e6fa51a9d43c7f/extensions/wikia/WikiFactory/WikiFactory.php#L571): The Cyclomatic Complexity of this function "setVarById" is 30 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: extensions/wikia/WikiFactory/WikiFactory.php [hunk 5]
```diff
@@ -1263,16 +1264,16 @@ static public function getLocalEnvURL( $url, $forcedEnv = null ) {
 		// we do not have valid ssl certificate for these subdomains
 		switch ( $environment ) {
```
Comment (WikiaTech) [inline extensions/wikia/WikiFactory/WikiFactory.php]: MAJOR Add a "case default" clause to this "switch" statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 118 issues
* ![MAJOR][MAJOR] 89 major
* ![MINOR][MINOR] 16 minor
* ![INFO][INFO] 13 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiFactory.php](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php): File "WikiFactory.php" has 3469 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS104)
1. ![MAJOR][MAJOR] [WikiFactory.php#L27](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php#L27): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L49](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php#L49): Class "WikiFactory" has 77 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [WikiFactory.php#L258](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php#L258): Reduce the number of returns of this function 6, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. ![MAJOR][MAJOR] [WikiFactory.php#L262](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php#L262): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L267](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php#L267): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L361](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php#L361): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L407](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php#L407): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L490](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php#L490): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiFactory.php#L571](https://github.com/Wikia/app/blob/46d6de6fa03aecd776e4f5e4704eb100596d8230/extensions/wikia/WikiFactory/WikiFactory.php#L571): The Cyclomatic Complexity of this function "setVarById" is 30 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[project-id]: wikia:app
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**